.github/workflows/lint.yml

```diff
@@ -0,0 +1,46 @@
+name: Linting
+
+env:
+  COMPOSER_VERSION: "2"
+  COMPOSER_CACHE: "${{ github.workspace }}/.composer-cache"
+
+on:
+  push:
+    branches:
+      - develop
+      - trunk
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  phpcs:
+    name: PHPCS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.4.0
+      - name: Set standard 10up cache directories
+        run: |
+          composer config -g cache-dir "${{ env.COMPOSER_CACHE }}"
+      - name: Prepare composer cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ env.COMPOSER_CACHE }}
+          key: composer-${{ env.COMPOSER_VERSION }}-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            composer-${{ env.COMPOSER_VERSION }}-
+      - name: Set PHP version
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.2'
+          coverage: none
+          tools: prestissimo, composer:v2
+      - name: composer install
+        run: composer install --ignore-platform-reqs
+      - name: Check PHPCS standard
+        run: ./vendor/bin/phpcs -i
+      - name: PHPCS check
+        uses: chekalsky/phpcs-action@v1
```

Comment:
We're already using [10up/phpcs-composer](https://github.com/10up/phpcs-composer) in the project. Would be great to keep tools consistent.

Refer [to this section](https://docs.google.com/document/d/1r6WSXn22bOYfotDd9oyqf_CPsPXBGK2M1TfKDeKQ7Ew/edit#heading=h.mhv3eyi68ryb) for more details.